pep-0001.txt

```diff
@@ -273,11 +297,19 @@ down by the Council, in which case a new BDFL-Delegate will be appointed in the
 same manner as for a new PEP (including Deferral of the PEP if no suitable
 replacement can be found).
 
-PEP review and resolution may also occur on a list other than python-dev
-(for example, distutils-sig for packaging related PEPs that don't
-immediately affect the standard library). In this case, the "Discussions-To"
-heading in the PEP will identify the appropriate alternative list where
-discussion, review and pronouncement on the PEP will occur.
+When the PEP process is being used for projects other than the CPython
+reference interpreter, PEP review and resolution may also occur on a list
+other than python-dev (for example, distutils-sig for packaging related PEPs
+that don't immediately affect the standard library). In this case, the
+"Discussions-To" heading in the PEP will identify the appropriate alternative
+list where discussion, review and pronouncement on the PEP will occur. The
+Steering Council may also choose to set up "standing delegations" for these
```

Comment:
Perhaps we should remove the standing delegation wording. To me, this is more an implementation detail than governance. For example, the Steering Council could, without additional wording here, delegate all packaging PEPs to one or two individuals until the end of the Steering Council term.

Reply:
How will we record the setting up of these delegation?  As @willingc says, the SC is within its authority to delegate that responsibility until the end of its term.  However *at* the end of the term, a potentially entirely new SC will come in.  They should have a definitive list of those delegations, which they could simply continue with an explicit declaration.  But that would be difficult if they didn't have that definitive list of delegations.

Reply:
@warsaw I think that the list could be maintained in the new "steering-council" repo which we will move from my account to python. In addition to a markdown list, it would also likely be captured in biweekly reports.

Reply:
Yeah, this is what I was referring to when I commented that I didn't think what I had here was actually enough to fix the gaps in the current standing delegation process, since we're still missing a record on the Python level governance side of the standing delegations that I listed in the PyPA specification management guidelines.

Between Carol's and Barry's comment at https://github.com/python/peps/pull/896#discussion_r257521315 though, I think this paragraph will be improved by making it *less* specific, and instead just reminding the Steering Council members that if we set something like this up, we should write it down somewhere where the next Steering Council can find it: https://github.com/python/peps/pull/896/commits/f851ef54bf6a4b8bf61f4314c7054be11d99f450

So the new wording just says that there should *be* records, but it doesn't attempt to specify what form those records will take.